oraibtcbridge/assetlist.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "oraibtcbridge",
+  "assets": [
+    {
+      "description": "The main native staking token of Orai Bitcoin Bridge",
+      "denom_units": [
+        {
+          "denom": "uoraibtc",
+          "exponent": 0
+        },
+        {
+          "denom": "ORAIBTC",
+          "exponent": 6
+        }
+      ],
+      "base": "uoraibtc",
+      "name": "oraibtcbridge",
+      "display": "ORAIBTC",
+      "symbol": "ORAIBTC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg"
+      },
+      "coingecko_id": "oraibtc-token",
```

Comment:
This coingecko id doesn't exist. Any coingecko id that doesn't exist must be removed (even if it will exist soon).